tap_hubspot/hubspot.py

```diff
@@ -297,10 +297,10 @@ def get_owners(self):
     def get_archived(self, object_type: str):
         path = f"/crm/v3/objects/{object_type}"
         data_field = "results"
-        filter_key = "archivedAt"
-        replication_path = [filter_key]
+        replication_path = ["archivedAt"]
+        properties = ["hs_object_id"]
```

Comment:
👍

Reply:
I meant we probably want to add something like this as a comment - this can seem like a very magical line for someone that hasn't been a part of this story.